### ISSUE TITLE
About applying CBP to other networks

### ISSUE THREAD

**Issue by yao0801** [gy20073/compact_bilinear_pooling#26]
Hi @gy20073 ,I can only find the Compact Bilinear Pooling applying in VGG16 network. So have you ever tried other networks. I try to use ResNet18 with learning rate of conv bn and scale layer setting to 0 before cbp layer during the 1st step of finetuning. But it seems not converge. So I wonder whether the lr of bn and scale layer should be fixed during the first step. Or this method cannot be applied to ResNet?

**Comment by abcdvzz** [comment]
> Hi @gy20073 ,I can only find the Compact Bilinear Pooling applying in VGG16 network. So have you ever tried other networks. I try to use ResNet18 with learning rate of conv bn and scale layer setting to 0 before cbp layer during the 1st step of finetuning. But it seems not converge. So I wonder whether the lr of bn and scale layer should be fixed during the first step. Or this method cannot be applied to ResNet?

Have u solved the problem? I tried to train the resnet model but it just cannot converge no matter how I refine the parameters.

**Comment by yao0801** [comment]
> > Hi @gy20073 ,I can only find the Compact Bilinear Pooling applying in VGG16 network. So have you ever tried other networks. I try to use ResNet18 with learning rate of conv bn and scale layer setting to 0 before cbp layer during the 1st step of finetuning. But it seems not converge. So I wonder whether the lr of bn and scale layer should be fixed during the first step. Or this method cannot be applied to ResNet?
> 
> Have u solved the problem? I tried to train the resnet model but it just cannot converge no matter how I refine the parameters.

I see the result in other paper training ResNet with CBP, but I cannot make it converge.
